documentation/Functions.md

```diff
@@ -152,6 +153,19 @@ npmInstall("my-thai-star/angular")
 
 ***
 
+### dockerCompose
+#### parameter 
+1. Path to the directory where the docker-compose.yml file is located, relative to workspace.
+2. Assertion information. Only needed for the console runner to check if the server was started properly.
+#### example 
+dockerCompose("my-thai-star", { "startupTime": 600, "port": 8081 })
+
+##### Assertion information
+startupTime = Time in seconds to wait before checking if the server is running
+port: Port on which the server is running
```

Comment:
Will this work for every project? I think in some cases we have to specify a path, too?

Reply:
a path to what exactly?